notebooks/working/3.0-mab-independent-variable-simulation.md

```diff
@@ -0,0 +1,1127 @@
+    alts_sim = np.random.choice(a=np.arange(series.max()+1),
+                                p=np.bincount(series)/len(series),
+                                size=av_size)
+
+    N = len(alt_name_dict)
+
+    av_sim = [np.array([1] * K + [0]*(N-K)) for K in alts_sim]
+
+    for x in av_sim:
+        np.random.shuffle(x)
+
+    np.random.shuffle(av_sim)
+    AV_columns = [alt_name_dict[i]+'_AV' for i in alt_name_dict.keys()]
+    AV_Df = pd.DataFrame(av_sim, columns=AV_columns)
+    choice = [random.choice(np.nonzero(a == 1)[0]) + 1 for a in np.array(AV_Df)]
+    choice_df = pd.DataFrame(choice, columns=['sim_choice'])
+    Sim_DF_AV = pd.concat([sim_data, AV_Df, choice_df], axis=1, sort=False)
+    return Sim_DF_AV
+```
+
+## Example Implementation 
+
+
+## Declaring variables
+
+```python
+observation_id_col = 'observation_id'
+
+alternative_id_col = 'mode_id'
+
+variable_type = {'num_kids': 'categorical',
+                 'household_size': 'categorical',
+                 'num_cars': 'categorical',
+                 'num_licensed_drivers': 'categorical'}
+
+individual_specific_variables = ['num_kids', 'household_size',
+                                 'num_cars', 'num_licensed_drivers']
+
+alternative_specific_variables = ['total_travel_time',
+                                  'total_travel_distance',
+                                  'total_travel_cost']
+
+trip_specific_variables = ['cross_bay']
+
+alternative_name_dict = {1: 'drive_alone',
+                         2: 'shared_2',
+                         3: 'shared_3p',
+                         4: 'wtw',
+                         5: 'dtw',
+                         6: 'wtd',
+                         7: 'walk',
+                         8: 'bike'}
+
+variable_type = {'num_kids': 'categorical',
+                 'household_size': 'categorical',
+                 'num_cars': 'categorical',
+                 'num_licensed_drivers': 'categorical',
+                 'cross_bay': 'categorical',
+                 'total_travel_time': 'continuous',
+                 'total_travel_distance': 'continuous',
+                 'total_travel_cost': 'continuous'}
+
+distributions = ['normal', 'alpha', 'beta', 'gamma', 'expon', 'gumbel']
+```
+
+## Implementation of Function
+
+```python
+bike_data_params = FindLongDataDist(data_long=bike_data_long,
+                                    alt_id_col=alternative_id_col,
+                                    obs_id_col=observation_id_col,
+                                    alt_spec=alternative_specific_variables,
+                                    alt_name_dic=alternative_name_dict,
+                                    ind_spec=individual_specific_variables,
+                                    trip_spec=trip_specific_variables,
+                                    var_types=variable_type,
+                                    cont_dists=distributions)
+```
+
+```python
+sim_data = SimDf(params_dict=bike_data_params, size=2000)
```

Comment:
Just from the look of the code itself, without checking the functions in detail, the difference between `sim_data` and `wide_sim_data` is not really clear.

Reply:
will do!